From 0619af197908bae3db519d89fd2447cb8aefe94c Mon Sep 17 00:00:00 2001
From: Ilias Apalodimas <ilias.apalodimas@linaro.org>
Date: Mon, 30 Nov 2020 11:47:41 +0200
Subject: [PATCH 1/1] cmd: efidebug: Add support for TCG2 final events table

A previous commit is adding EFI_TCG2_PROTOCOL, which in it's eventlog
support registers an EFI configuration table.
Let's add the necessary GUID so 'efidebug table' command can display
table names properly.

Signed-off-by: Ilias Apalodimas <ilias.apalodimas@linaro.org>
---
 cmd/efidebug.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/cmd/efidebug.c b/cmd/efidebug.c
index 5288b9920b..70bb73ba02 100644
--- a/cmd/efidebug.c
+++ b/cmd/efidebug.c
@@ -278,6 +278,10 @@ static const struct {
 		"Runtime properties",
 		EFI_RT_PROPERTIES_TABLE_GUID,
 	},
+	{
+		"TCG2 Final Events Table",
+		EFI_TCG2_FINAL_EVENTS_TABLE_GUID,
+	},
 };
 
 /**
-- 
2.29.2

